source/geometry_model/chunk.cc

```diff
@@ -469,6 +729,10 @@ namespace aspect
     aspect::Utilities::Coordinates::CoordinateSystem
     Chunk<dim>::natural_coordinate_system() const
     {
+      // TODO This will give problems somewhere down the line
```

Comment:
- At the moment there is no problem as there are no calls to natural_coordinate_system anywhere. 
- There will only be a problem when after a call to natural_coordinate_system that returns `CoordinateSystem::Spherical`, the `Utilities::Coordinates::cartesian_to_spherical` function is called instead of `cartesian_to_natural_coordinates` function because of the different interval for longitude and the use of latitude instead of colatitude. 
- I think I mostly find it inconsistent to call the natural coordinate system `CoordinateSystem::Spherical` while it is not (i.e. different longitude interval and latitude). Perhaps we need to introduce another `CoordinateSystem`?

Reply:
I see. What about `CoordinateSystem::Geographical`? Anyway if it works for now we can also leave the TODO.